src/Umbraco.Cms.Integrations.Script.Cookiebot/build/Umbraco.Cms.Integrations.Script.Cookiebot.targets

```diff
@@ -0,0 +1,29 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<PropertyGroup>
+		<CookiebotPropertyEditorFilesPath>
+			$(MSBuildThisFileDirectory)..\Views\Partials\UmbracoCms.Integrations\Scripts\Cookiebot\**\*.*
+		</CookiebotPropertyEditorFilesPath>
+	</PropertyGroup>
+	<Target Name="CopyCookiebotPropertyEditorAssets" BeforeTargets="Build">
```

Comment:
As noted you can likely remove most or all of this if using the Razor class library approach.